src/pages/index.astro

```diff
@@ -36,4 +55,130 @@ import HeroCTA from "../features/landing/sections/HeroCTA.astro"
     left: -275px;
     overflow: hidden;
   }
+
+  .heroContainer {
+    padding-top: var(--space-24x);
+  }
+
+  .hero {
+    max-width: var(--max-width);
+    margin: 0 auto;
+    padding-left: var(--space-6x);
+    padding-right: var(--space-6x);
+  }
+
+  .hero h2 {
+    font-family: var(--font-family-text);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 56px;
+    line-height: 61.6px;
+    display: flex;
+    align-items: center;
+    letter-spacing: -2.7px;
+    margin-bottom: var(--space-8x);
+    color: #0c162c;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .recommended {
+    max-width: var(--max-width);
+    padding-left: var(--space-6x);
+    padding-right: var(--space-6x);
+  }
+
+  .recommended h4 {
+    color: var(--color-gray-600);
+    font-size: var(--space-3);
+    text-transform: uppercase;
+    letter-spacing: 1.28px;
+    font-weight: var(--font-weight-medium);
+  }
+
+  .recommended h5 {
+    color: var(--color-gray-600);
+    font-size: var(--space-3);
+    font-weight: var(--font-weight-medium);
+    text-transform: uppercase;
+    letter-spacing: 1.28px;
+    margin-top: var(--space-16x);
+    font-family: var(--font-family-text);
+    display: block;
+  }
+
+  .recommended h3 {
+    color: var(--gray-900);
+    font-size: var(--space-8x);
+    letter-spacing: -0.96px;
+    font-weight: var(--font-weight-medium);
+  }
+
+  .recommended div {
+    display: flex;
+    flex-wrap: wrap;
+    gap: var(--space-4x);
+    margin-top: var(--space-8x);
+  }
+
+  .recommended a {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: var(--space-4x) var(--space-5x);
+    box-shadow: 0px var(--space-5x) var(--space-8x) calc(-1 * var(--space-5x)) rgba(12, 22, 44, 0.24);
```

Comment:
the shadow is "12px" not "20x"

`box-shadow: 0px 12px 32px -12px #0C162C1F;`

Reply:
It didn't look the same as Figma so I tried to add few pixels but I'll revert it back